build/azure-pipelines/ci-build-test.yml

```
@@ -0,0 +1,152 @@
+    exclude:
+      - 'src/services/**/*.md'
+      - 'src/services/**/README.md'
+
+variables:
+  buildConfiguration: 'Release'
+  solution: 'src/BankSystem.sln'
+  testProjectsPattern: 'src/**/tests/**/*.csproj'
+  resultsDirectory: '$(Agent.TempDirectory)/TestResults'
+
+stages:
+- stage: BuildAndTest
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 0

(indentation)